app/routes/users/components/UserProfile.js

```diff
@@ -312,6 +312,66 @@ export default class UserProfile extends Component<Props, EventsProps> {
                 </Card>
               </div>
             )}
+            {/* canChangeGrade is a good heuristic if we should show permissions. All users can see their own permission via the API, but only admins can show permissions for other users.*/}
```

Comment:
newline after period ⭕️  this won't fit in my terminal 💯

Reply:
:100: